velero-csi-plugin/volumesnapshot_restorer.go

```diff
@@ -39,15 +39,14 @@ func (p *VSRestorer) AppliesTo() (velero.ResourceSelector, error) {
 
 func (p *VSRestorer) Execute(input *velero.RestoreItemActionExecuteInput) (*velero.RestoreItemActionExecuteOutput, error) {
 	p.log.Info("Starting VSRestorer")
-	var vs snapshotv1alpha1api.VolumeSnapshot
+	var vs snapshotv1beta1api.VolumeSnapshot
 
 	if err := runtime.DefaultUnstructuredConverter.FromUnstructured(input.Item.UnstructuredContent(), &vs); err != nil {
 		return &velero.RestoreItemActionExecuteOutput{}, err
 	}
 
-	// Source is the PVC from which a snapshot was created.
-	// This must be removed so that the controller doesn't think we're trying to create a new snapshot
-	vs.Spec.Source = nil
+	// This section of code used to set the VolumeSnapshot.Spec.Source field to nil
```

Comment:
I think we can just remove this comment, but not blocking on it, can always do it in a subsequent PR.

Reply:
Yeah, this was more a reminder for me as we move forward - I think it'll need to be fixed, but didn't have the exact fix on hand yet.